torch/csrc/jit/fuser/executor.cpp

```diff
@@ -385,7 +385,7 @@ bool runFusion(const int64_t key, Stack& stack, std::string* code_out) {
     if (omap.needsSumToSize()) {
       return at::sum_to(
           raw_outputs[omap.offset()],
-          all_inputs[omap.sizeInput()].toIntList()->elements());
+          c10::impl::toArrayRef(all_inputs[omap.sizeInput()].toIntList()->elements()));
```

Comment:
this is weirdly named, why can't we have a shorter version of this?

Reply:
like? `toArrayRef` is pretty short.

Reply:
`toArrayRef` is gone